Homepage update fails with newer npm version

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Homepage

### 📂 What was the exact command used to execute the script?

update

### ⚙️ What settings are you using?

- [ ] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 📝 Provide a clear and concise description of the issue.

Homepage update fails when npm is updated before.

### 🔄 Steps to reproduce the issue.

npm install -g npm
npm install -g pnpm
update

See below error output.

### ❌ Paste the full error output (if available).

```
    __  __                                          
   / / / /___  ____ ___  ___  ____  ____ _____ ____ 
  / /_/ / __ \/ __ `__ \/ _ \/ __ \/ __ `/ __ `/ _ \
 / __  / /_/ / / / / / /  __/ /_/ / /_/ / /_/ /  __/
/_/ /_/\____/_/ /_/ /_/\___/ .___/\__,_/\__, /\___/ 
                          /_/          /____/       

  ⏳   Updating Homepage to v1.4.0 (Patience)Lockfile is up to date, resolution step is skipped
Packages: +17 -1
+++++++++++++++++-
Progress: resolved 17, reused 7, downloaded 10, added 11, done

dependencies:
- luxon 3.5.0
+ luxon 3.6.1
- react-i18next 11.18.6
+ react-i18next 15.5.3
- systeminformation 5.25.11
+ systeminformation 5.27.7

devDependencies:
- eslint-plugin-prettier 5.2.6
+ eslint-plugin-prettier 5.5.1
- eslint-plugin-react-hooks 5.1.0
+ eslint-plugin-react-hooks 5.2.0
- prettier 3.5.2
+ prettier 3.6.2
- tailwind-scrollbar 4.0.1
+ tailwind-scrollbar 4.0.2

Done in 1.3s using pnpm v10.8.1
npm error code EBADDEVENGINES
npm error EBADDEVENGINES The developer of this package has specified the following through devEngines
npm error EBADDEVENGINES Invalid engine "packageManager"
npm error EBADDEVENGINES Invalid name "pnpm" does not match "npm" for "packageManager"
npm error EBADDEVENGINES {
npm error EBADDEVENGINES   current: { name: 'npm', version: '11.4.2' },
npm error EBADDEVENGINES   required: { name: 'pnpm', version: '10.8.1' }
npm error EBADDEVENGINES }
npm error A complete log of this run can be found in: /root/.npm/_logs/2025-07-16T13_11_26_446Z-debug-0.log

[ERROR] in line 59: exit code 0: while executing command $STD npx --yes update-browserslist-db@latest

```

### 🖼️ Additional context (optional).

Updated as described here https://github.com/community-scripts/ProxmoxVE/discussions/561

--- a/ct/homepage.sh
+++ b/ct/homepage.sh
@@ -56,7 +56,7 @@ function update_script() {
     rm -rf homepage-${RELEASE}
     cd /opt/homepage
     $STD pnpm install
-    $STD npx --yes update-browserslist-db@latest
+    $STD pnpm update --no-save caniuse-lite
     export NEXT_PUBLIC_VERSION="v$RELEASE"
     export NEXT_PUBLIC_REVISION="source"
     export NEXT_PUBLIC_BUILDTIME=$(curl -fsSL https://api.github.com/repos/gethomepage/homepage/releases/latest | jq -r '.published_at')

